Source/WebCore/rendering/RenderElementInlines.h

```diff
@@ -98,11 +98,15 @@ inline bool RenderElement::shouldApplyPaintContainment() const
 
 inline bool RenderElement::shouldApplySizeContainment() const
 {
-    return shouldApplySizeOrStyleContainment(style().containsSize() || style().contentVisibility() == ContentVisibility::Hidden);
+    if (shouldSkipContent())
+        return true;
+    return shouldApplySizeOrStyleContainment(style().containsSize());
 }
 
 inline bool RenderElement::shouldApplySizeOrInlineSizeContainment() const
 {
+    if (shouldSkipContent())
+        return true;
     return shouldApplySizeOrStyleContainment(style().containsSizeOrInlineSize() || style().contentVisibility() == ContentVisibility::Hidden);
```

Comment:
Since `shouldSkipContent()` already checks for C-V hidden, this works too:
```suggestion
    return shouldSkipContent() || shouldApplySizeOrStyleContainment(style().containsSizeOrInlineSize());
```

Reply:
Done/